python/pyarrow/tests/test_plasma.py

```diff
@@ -755,6 +755,18 @@ def test_use_one_memory_mapped_file(self):
             create_object(self.plasma_client, DEFAULT_PLASMA_STORE_MEMORY + 1,
                           0)
 
+    def test_use_huge_pages(self):
+        subprocess.check_call("sudo mkdir -p /mnt/hugepages", shell=True)
```

Comment:
Please don't do `sudo` calls in unit tests. This could rather be a script that is run as an extra step in the CI. Still, this does not seem to test any Arrow/Plasma functionality?

Reply:
The filesystem created here is used to test Plasma support for huge pages, which is a performance optimization.